### PR TITLE
V0.3.0

### PR DESCRIPTION
**Pull request by alex-quiterio**

### CODE REVIEW

File: lib/packlink_lite/configuration.rb [hunk 1]
```diff
@@ -2,6 +2,11 @@ module PacklinkLite
   class Configuration
     attr_accessor :api_key, :testing, :timeout
     attr_writer :api_endpoint
+    attr_reader :headers
+
+    def initialize
+      @headers = { 'Content-Type' => 'application/json' }
```
Comment (coalwater) [inline lib/packlink_lite/configuration.rb]: The original headers had an 'Accept' header, but you replaced it with 'Content-Type'

File: lib/packlink_lite/configuration.rb [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
-    alias_method :testing?, :testing
+    attr_writer :api_endpoint
+    attr_reader :headers
+
+    def initialize
+      @headers = { 'Content-Type' => 'application/json' }
+    end
+
+    def api_endpoint
+      @api_endpoint || default_endpoint
```
Comment (fteem) [inline lib/packlink_lite/configuration.rb]: So, if I set a `api_endpoint` I will hit production when `testing?` is `true`?

Comment (alex-quiterio) [reply lib/packlink_lite/configuration.rb]: The point of using `default_endpoint` is just to keep the old behaviour unchanged. If we set the `api_endpoint`, we should change it for the test context as well.

File: lib/packlink_lite/configuration.rb (hunk 1 above)
Comment (alex-quiterio) [reply lib/packlink_lite/configuration.rb]: btw, this doesn't affect how the request is performed. But yes, you have a point on this one. Anyway, this is not a blocker for the deployment of `cw-shipping`

**Comment by fteem** [comment]
LGTM 😞 

(Looked Good To Me)

**Pull request closed without merging**